package.json

```diff
@@ -71,7 +71,7 @@
     "@babel/preset-env": "^7.2.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.1.0",
-    "@babel/register": "^7.0.0",
+    "@babel/register": "^7.4.4",
```

Comment:
Is there something specific to 7.4.4 and above that is needed for this PR?

Reply:
No, I'll remove it.